app/presenters/publishing_api/organisation_presenter.rb

```diff
@@ -96,7 +97,7 @@ def summary
 
```

Comment:
I don't suppose you could Govspeak render the `summary` in this too could you?  We'll only need to republish :-(

Reply:
Done